src/frontends/paddle/src/internal/pass/transform_fakequantize.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "internal/pass/transform_fakequantize.hpp"
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/variant.hpp>
+
+#include "default_opset.hpp"
+#include "openvino/pass/pattern/op/label.hpp"
+
+using namespace ov::frontend::paddle::op::default_opset;
+using namespace ov;
+using namespace ov::pass;
+using namespace ov::frontend::paddle::op;
+
+/*
+                                                                       x
+                                                                   |  |  |  |
+                                                                   |  |  |  |
+                                 /                                 V  v  v  v
+                                |              +-----------+     +-----------+
+                                |   scale -->  | Multiply  | --> |  Divide   |
+    +------------------+        |              +-----------+     +-----------+
+    |                  |        |                                      |
+    |  quantize_linear |  -->   |                                      v
+    |                  |        |                                +-----------+
+    +------------------+        |                                |   Round   |
+                                |                                +-----------+                  |   |   |   |
+                                |                                      |                        v   v   v   v
+                                |                                      v                    +-------------------+
+                                |                                +-----------+              |                   |
+                                |                                |   Clamp   |     === >    |   FakeQuantize    |
+                                 \                               +-----------+              |                   |
+                                                                       |                    +-------------------+
+                                 /                                     v                        |   |   |   |
+     +------------------+       |                                +-----------+                  v   v   v   v
+     |                  |       |                                |  Convert  |
+     |dequantize_linear |  -->  |                                +-----------+
+     |                  |       |                                      |
+     +------------------+       |                                      v
+                                |              +-----------+     +-----------+
+                                |   scale -->  | Multiply  | --> | Multiply  |
```

Comment:
The process of "dequantize" is convert-substract-multiply. Would you please explain the comments here?